Documentation/data-format.md

```diff
@@ -0,0 +1,69 @@
+# Adobe Experience Platform Edge Bridge for iOS
+
+## Data Format
+
+The AEP Edge Bridge extension maps the data sent in `trackAction` and `trackState` API calls to a specific format known to Analytics which requires no additional server-side mapping. This document outlines the data format applied to the tracking data.
+
+
+As of version 5.0.0 of AEP Edge Bridge for iOS, the following table lists the mapping of the `trackAction` and `trackState` parameters to the "data" element of the Experience Event sent to the Edge Network.
+
+
+| Data | Key in dispatched request | Comments |
```

Comment:
Key in dispatched request -> Key path in the network request
Comments -> Description